src/commands/generateHttpFile.ts

```diff
@@ -118,12 +116,22 @@ function parseBody(requestBody: OpenAPIV3.RequestBodyObject | undefined): string
         let body: { [key: string]: any } = {};
         if (jsonBodySchema.example) {
             body = jsonBodySchema.example;
+        } else if (jsonBodySchema.default) {
```

Comment:
why example prior to default?

Reply:
An example illustrates what the value is supposed to be. A default value is what the server uses if the client does not provide the value.
Refer to https://swagger.io/docs/specification/adding-examples/#:~:text=An%20example%20illustrates%20what%20the,does%20not%20provide%20the%20value.

The `example` would be a little more real. So use `example` first.